gui/slick/interfaces/default/config_postProcessing.tmpl

```diff
@@ -71,7 +72,12 @@
                                 <span class="component-desc">
                                     <select name="process_method" id="process_method" class="form-control input-sm">
                                         #set $process_method_text = {'copy': "Copy", 'move': "Move", 'hardlink': "Hard Link", 'symlink' : "Symbolic Link"}
-                                        #for $curAction in ('copy', 'move', 'hardlink', 'symlink'):
+                                        #set $process_action = ('copy', 'move', 'hardlink', 'symlink')
```

Comment:
@fernandog You should do a if ... else instead of set and if.
```
#if sys.platform == 'win32':
    #set $process_method = ('copy', 'move')
#else
    #set $process_method = ('copy', 'move', 'hardlink', 'symlink')
#end if
#for $curAction in $process_method: